csharp-basic-topics/ZipFilesInNet/ZipFilesInNET/ZipFilesInNET.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <RootNamespace>Zip_files_in_NET</RootNamespace>

```

Comment:
We've still got a reference to the old naming - this should be removed and the namespace/usings updated in any necessary files.